docs/configuration.md

```diff
@@ -1,108 +1,114 @@
+    ```
+    redis-server --loadmodule /path/to/redisgears.so <option> <value> ...
+    ```
 
-From redis-cli:
+    For setting the module's configuration options in with .conf file use the following format:
 
-```
-127.0.0.6379> MODULE load redisgears.so OPT1 OPT2
-```
+    ```
+    loadmodule /path/to/redisgears.so <option> <value> ...
+    ```
 
-From command line:
+    For setting the module's configuration with the [`MODULE LOAD`](https://redis.io/commands/module-load) command use:
 
-```
-$ redis-server --loadmodule ./redisgears.so OPT1 OPT2
-```
+    ```
+    127.0.0.1:6379> loadmodule /path/to/redisgears.so <option> <value> ...
+    ```
 
-## Passing configuration options at runtime
+**Runtime Configuration**
 
-It is possible to modify certain configuration parameters at runtime using `RG.CONFIGSET` command. The command receives the configuration parameter's name and its value. For example, the following will enable the excution profiler:
-```
-$ RG.CONFIGSET ProfileExecutions 1
-```
+Some configuration options may be set at runtime. Refer to each option's description for runtime configurability.
+
+!!! abstract "Related commands"
+    The following RedisGears commands are related to configuration:
 
-## RedisGears configuration options
+    * [`RG.CONFIGGET`](commands.md#rgconfigget)
+    * [`RG.CONFIGSET`](commands.md#rgconfigset)
 
 ## MaxExecutions
+The **MaxExecutions** configuration option controls the maximal number of executions that will be saved in the executions list. Once this threshold value is reached, older executions will be deleted from the list by order of their creation (FIFO). Only executions with a 'done' status (?) are deleted.
+
+_Expected Value_
 
-The maximum amount of execution to save. When reach this number, old execution will be deleted in a FIFO order. Notice that only the execution that has been finished will be deleted (pending execution will be deleted on done).
+Integer
 
-### Default
+_Default Value_
 
 1000
 
-### Configurable at Runitime
+_Runtime Configurability_
 
-** Yes
+Supported
 
-### Example
+**Examples**
 
 ```
-$ redis-server --loadmodule ./redisearch.so MaxExecutions 10
+127.0.0.1:6379> RG.CONFIGSET MaxExecutions 10
+OK
 ```
 
----
-
 ## MaxExecutionsPerRegistration
+The **MaxExecutionsPerRegistration** configuration option controls the maximal number of executions that are saved in the list per registration. Once this threshold value is reached, older executions for that registration will be deleted from the list by order of their creation (FIFO). Only executions with a 'done' status (?) are deleted.
+
+_Expected Value_
 
-The maximum amount of execution to save per registration. When reach this number, old execution will be deleted in a FIFO order. Notice that only the execution that has been finished will be deleted (pending execution will be deleted on done).
+Integer
 
-### Default
+_Default Value_
 
 100
 
-### Configurable at Runitime
+_Runtime Configurability_
 
-** Yes
+Supported
```

Comment:
Will take effect only at the next execution creation..

Reply:
Fixed via 079f280